src/DisplayTotalMasteryPage/DisplayTotalMasteryPage.tsx

```diff
@@ -0,0 +1,92 @@
+import { Container, makeStyles, Theme, Typography } from '@material-ui/core'
+import React, { FC, useContext, useState } from 'react'
+import { useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import { GlobalContext } from '../GlobalContext'
+import { theme } from '../theme'
+
+interface DisplayTotalMasteryPageProps {}
```

Comment:
Let's shorten this a bit

```suggestion
interface MasteryPageProps {}
```